src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { PeopleTable } from '../../components/PeopleTable/PeopleTable';
+
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+export const PeoplePage: React.FC = () => {
+  return (
+    <>
+    <h1>People Page</h1>
+    <PeopleTable />
+    </>
+  )
+}
```

Comment:
```suggestion
export const PeoplePage: React.FC = () => (
  <>
    <h1>People Page</h1>
    <PeopleTable />
  </>
  )
```
no need to write return, just use round brackets